### ISSUE TITLE
Vorta backups are run twice when using scheduled backups

### ISSUE THREAD

**Issue by DanHam** [borgbase/vorta#1095]
**Describe the bug**
I believe backups are *sometimes* being run twice when using scheduled backups. 

This *sometimes* results in Vorta reporting that a backup did not complete successfully (one of the two duplicate scheduled backups fails).
This *sometimes* results in Vorta creating two backups for the same scheduled backup - these can be seen in the 'Archives' tab .

![Screenshot from 2021-11-03 19-23-55](https://user-images.githubusercontent.com/4364798/140179016-7cfb86ab-0055-4a1e-9ba6-2e045d8e0f96.png)

In addition to the above issues, the value for 'Next Backup' in the 'Schedule' tab does not appear to be updated - clearly this is fairly minor.

**To Reproduce**
1. Use the 'Backup every...' facility in the Schedule tab

**Environment (please complete the following information):**
- OS: Arch Linux
- Vorta version: Vorta 0.7.8
- Installed from: Arch AUR - vorta-git. (`pacman -Q vorta-git` outputs `vorta-git r565.fba6a2f-1`)
- Borg: 1.1.17

**Additional context**
I previously used the [vorta](https://aur.archlinux.org/packages/vorta/) AUR package. However, I was affected by #1083 and switched to using [vorta-git](https://aur.archlinux.org/packages/vorta-git/). As such I am running Vorta built from the latest commit - currently `fba6a2f`.

[Vorta logs HERE](https://gist.github.com/DanHam/4ebf7c09811db4cdd66008234b5226d3)

If you need me to run Vorta in debug mode please let me know (and tell me how to do it!). If you need any further info please let me know.

**Comment by m3nu** [comment]
The scheduler from #1086  is still very new, but fairly simple. I'll look over it again.

**Comment by bastiencyr** [comment]
Hey @m3nu, 

I think that the error comes from refreshing : 

```
2021-11-02 19:38:48,534 - vorta.scheduler - DEBUG - Refreshing all scheduler timers
2021-11-02 19:38:48,547 - vorta.scheduler - INFO - Setting timer for profile 5
2021-11-02 19:38:48,550 - vorta.scheduler - DEBUG - Scheduling next run for 2021-11-02 20:01:00
2021-11-02 19:53:48,578 - vorta.scheduler - DEBUG - Refreshing all scheduler timers
2021-11-02 19:53:48,591 - vorta.scheduler - INFO - Setting timer for profile 5
2021-11-02 19:53:48,593 - vorta.scheduler - DEBUG - Scheduling next run for 2021-11-02 20:01:00
```
It adds a job thanks to the jos manager every 15 minutes and sometimes fails because the archive name is the same. 
But it can add a job as many times as the scheduler wants since the jobs manager doesnt limit jobs. But we said in #1086 that the jobs scheduler has to limit number of jobs per profile to a single one. So i think you can assign this issue to me unless im wrong

**Comment by m3nu** [comment]
You're right. I was thinking the same shortly after I responded. Would be great if you can look into it and combine it with the planned jobs scheduler enhancement. 🙏

**Comment by m3nu** [comment]
> the value for 'Next Backup' in the 'Schedule' tab does not appear to be updated - clearly this is fairly minor.

I looked into this, but couldn't get the signals working correctly in a simple way. Might try again.

> It adds a job thanks to the jos manager every 15 minutes and sometimes fails because the archive name is the same.

I'm trying to add a lock here. Might help. You should still check if this fixes it.

**Comment by m3nu** [comment]
Should be mostly solved in #1096. Looks correct after doing backups every minute for a while.

![Screen Shot 2021-11-06 at 10 46 36](https://user-images.githubusercontent.com/3916435/140600679-a75026e3-c675-4725-9efb-a7c5841df985.jpg)

**Comment by DanHam** [comment]
@m3nu @bastiencyr 

Many thanks for taking the time to look into this and getting a fix in place. Much appreciated.

I'll be sure to update over the next couple of days.
